bower.json

```diff
@@ -39,9 +38,9 @@
     "*.md"
   ],
   "resolutions": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
   },
   "devDependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
The pieChart padAngle and cornerRadius options.  Using the option on a version of d3 that doesn't have the capability should just do nothing.